provisioning/roles/mangaki_back/tasks/main.yml

```diff
@@ -1,4 +1,26 @@
 ---
+- name: Setup Redis.
+  include_role:
+    name: utils/redis
+  vars:
+    redis_databases: '{{ mangaki_back_redis_database + 1 | int }}'
```

Comment:
Right, thank you. `mangaki_back_redis_database` is 0-indexed, `redis_databases` is the amount of database created by Redis (localhost/0, localhost/1, …, localhost/{{redis_databases - 1}}).

Though, I honestly think that this piece of code is stupid and we should use everytime 0 as default database, because even if we were to use, let's say, database 3 for Mangaki beta and 2 for tests, then 2 would have to be deployed before 3 otherwise 2 would overwrite 3's configuration of Redis and destroy the /2 database.

That's really a WIP for this part.

A part of me wants to support non-destructive Redis reconfiguration.